internal_packages/keybase/lib/modal-key-recommender.cjsx

```diff
@@ -68,8 +68,23 @@ class ModalKeyRecommender extends React.Component
         ['Encrypt', 'Cancel']
       )
         return
-    Actions.closePopover()
-    @props.callback(@state.identities)
+
+    if _.every(@state.identities, (identity) -> identity.key?)
+      Actions.closePopover()
+      @props.callback(@state.identities)
+    else
+      emptyIdents = _.filter(@state.identities, (identity) -> !identity.key?)
```

Comment:
Might make things slightly cleaner to bump this up a line and then say `if emptyIdents.length === 0` instead of _.every!